skyportal/facility_apis/observation_plan.py

```diff
@@ -551,14 +557,25 @@ def generate_plan(
             field_ids = {}
             if use_skyportal_fields is True:
                 for request in requests:
-                    field_tiles_query = sa.select(InstrumentField.field_id).where(
-                        LocalizationTile.localization_id == request.localization.id,
-                        LocalizationTile.probdensity >= min_probdensity,
-                        InstrumentFieldTile.instrument_id == request.instrument.id,
-                        InstrumentFieldTile.instrument_field_id == InstrumentField.id,
-                        InstrumentFieldTile.healpix.overlaps(LocalizationTile.healpix),
+                    query_id = (
+                        f"{str(request.localization.id)}_{str(request.instrument.id)}"
```

Comment:
What happens if between requests to the same instrument the user changed the integrated probability?

Reply:
Agreed. I think we want to cache by localization id, instrument id AND cumprob.

Reply:
@Theodlz Sounds good. Let's try this after Guy's PR